bubble-wrap-style.yaml

```diff
@@ -1382,11 +1431,111 @@ layers:
             text-blend-order:
                 text_source: global.ux_language_text_source
                 visible: false    # labels are enabled by each layer below
+                priority: |
+                    function() { return (feature['min_zoom'] + (1 - 1 / feature['collision_rank']) * 0.1) || 65; }
                 font:
                     family: global.text_font_family
                     weight: normal
                     fill: global.text_fill
 
+#        z_debug:
+#           filter: { $geometry: line, kind: true, $zoom: { min: 1 }, source: [naturalearthdata.com, openstreetmap.org] }
+#           draw:
+#               lines:
+#                   color: |
+#                      function() {
+#                          var letters = '0123456789ABCDEF';
+#                          var color = '#';
+#                          for (var i = 0; i < 6; i++ ) {
+#                              color += letters[Math.floor(Math.random() * 16)];
+#                          }
+#                          return color;
+#                      }
+#                   order: 1000
+#                   width: 1px
+#                   visible: true
+
+        hgv-interlay:
+            filter:
+                all:
+                    - hgv: true
+                    - function() { return global.sdk_hgv_overlay; }
+            draw:
+                lines:
+                    visible: true
+                    color: blue
+                    width: 1.7px
+            z-none:
+                filter:
+                    - hgv_restriction: true
+                draw:
+                    lines:
+                        color: red
+                        width: 2.5px
+            z-restricted:
+                filter:
+                    - hgv_restriction: true
+                draw:
+                    lines:
+                        color: purple
+                        width: 2.5px
+                tunnel:
+                    filter: {is_tunnel: true, $zoom: {min: 13} }
+                    draw:
+                        lines:
+                            # color: [[16, [0.843,0.843,0.843]]]
+                            color: purple
```

Comment:
This was needed because it "folds" in with the general roads layer styling which has a more specific selector for road tunnels which reset the purple coloring in this layer's parent. Fun times.